package.json

```diff
@@ -8,10 +8,12 @@
     "@testing-library/jest-dom": "5.11.4",
     "@testing-library/react": "11.1.0",
     "@testing-library/user-event": "12.1.10",
+    "@types/axios": "0.14.0",
```

Comment:
`Axios` already comes with the template. Why do we need to add it again? And this package is deprecated as `axios` provides it's own type definition.

Reply:
Good catch. Fixed in b086a7f5c1d90bebebdfcf18c952bdcadcacba08